Dockerfile

```diff
@@ -17,10 +17,10 @@ RUN apk add --no-cache \
       build-base \
       libffi-dev
 # Install Radicale
-ADD https://github.com/Kozea/Radicale/archive/${VERSION}.tar.gz radicale.tar.gz
-RUN tar xzf radicale.tar.gz && \
+RUN wget --quiet https://github.com/Kozea/Radicale/archive/${VERSION}.tar.gz && \
```

Comment:
GitHub sends the ``Content-Disposition`` HTTP header with an alternative filename:
```
Content-Disposition: attachment; filename=Radicale-master.tar.gz
```
At the moment **wget** has only experimental support for this and it's disabled by default. Once it gets enabled however, the file will be saved as as ``Radicale-${VERSION}.tar.gz`` instead of ``${VERSION}.tar.gz`` and the command will fail.
You should specify the target filename explicitly with something like ``-O radicale.tar.gz``.